doc/2d_combine.rst

```diff
@@ -0,0 +1,103 @@
+.. _2d_combine:
+
+=========================
+Combine Science Exposures
+=========================
+
+Overview
+========
+
+PypeIt can combine (without weighting) multiple science exposures
+as part of the data reduction process. See :ref:`coadd2d` for a weighted
+coadd, which is done outside of the data reduction pipeline.
+
+To combine (without weighting) multiple science exposures, the user needs to edit
+the :ref:`pypeit_file` according to the desired reduction.
+The process to combine multiple science exposures is basically identical
+to :ref:`a-b_differencing` without the subtraction of a background frame.
+
+PypeIt file
+===========
+
+Data Block
+----------
+After running :ref:`pypeit_setup`, the user should update the
+:ref:`pypeit_file:Data Block` in the PypeIt file to add three extra
+columns, and name them ``calib``, ``comb_id`` and ``bkg_id``.
+This can also be obtained by running :ref:`pypeit_setup` and adding the `-b` option.
+The columns ``calib`` and ``comb_id`` should be edited according to the desired reduction,
+while ``bkg_id`` is not used here and its value should be set to -1.
+
+Parameter Block
+---------------
+The combining process is guided by the parameters in :ref:`pypeit_par:ProcessImagesPar Keywords`.
+Currently, there are only 2 available methods to combine multiple frames: ``median`` and  ``weightmean``.
+The default is ``weightmean``. The user can change this parameter in the PypeIt file in
+the following way ::
```

Comment:
Maybe clarify what "weightmean" does here, which I believe is an unweighted coadd, but which uses the mask information, i.e. this will appropriately mask and average images with cosmics or other bad pixels (saturation?), but it does not do a weighted mean in any other way. Otherwise, "weightmean" seems to contradict the statement above about this not being a weighted coadd.

Reply:
Make sense! Done!